android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationListenerService.java

```diff
@@ -93,6 +98,14 @@ private void sendNotification(ReactApplicationContext context, Bundle bundle) {
         if (bundle.getString("contentAvailable", "false").equalsIgnoreCase("true")) {
             jsDelivery.notifyRemoteFetch(bundle);
         }
+
+        Log.e(LOG_TAG, "RNPushNotificationListenerService.sendNotification: " + bundle);
+
+        if (!isRunning) {
```

Comment:
This only triggers if the app is not running. So an app in the foreground, it's the responsibility of the client JS code to present a notification (or in-app display, right?) What about if the app is running in the background where there is no in-app display to show it...this change will still not deliver it to the notification center. Is that desired? (Sorry, I'm a bit unclear on what the behavior used to be.)